gst/gst_message.go

```diff
@@ -357,13 +357,13 @@ type QoSValues struct {
 	// If the message was generated by a live element
 	Live bool
 	// The running time of the buffer that generated the message
-	RunningTime time.Duration
+	RunningTime time.Duration // should this also be a ClockTime or even just uint64?
```

Comment:
in the [docs](https://gstreamer.freedesktop.org/documentation/gstreamer/gstmessage.html?gi-language=c#gst_message_parse_qos) they are uint64, but I have not worked with messages yet. 

There are more fields on that struct that are currently durations: running_time, stream_time, timestamp, duration

I am trying to remove the old duration stuff as much as possible before we commit to a semver tagged release, to prevent a breaking change

Reply:
I'll rephrase: do you think we should change some of them to uint64 or can all of them stay as `time.Duration`? I agree that `ClockTime` is the wrong type here

Reply:
To be honest, I'm not sure. I _guess_ a RunningTime is a Duration by definition right.

Reply:
okay in that case I'll leave the comments in place as a reminder and not change anything for now.